frontend/src/metabase/lib/query/query.js

```diff
@@ -180,9 +180,23 @@ function setBreakoutClause(query: SQ, breakoutClause: ?BreakoutClause): SQ {
     .map(b => FIELD_REF.getFieldTargetId(b))
     .filter(id => id != null);
   for (const [index, sort] of getOrderBys(query).entries()) {
-    const sortId = FIELD_REF.getFieldTargetId(sort[1]);
-    if (sortId != null && !_.contains(breakoutIds, sortId)) {
-      query = removeOrderBy(query, index);
+    const sortField = sort[1];
+    const sortId = FIELD_REF.getFieldTargetId(sortField);
+    if (sortId != null) {
+      // Remove invalid field reference
+      if (!_.contains(breakoutIds, sortId)) {
```

Comment:
Should we use `breakoutIds.includes(sortId)` here maybe?

Reply:
Good point. This'll break IE, but I think we're good with that.